concepts/80 Troubleshooting/30 3rd-Party Frameworks Issues/00 Bootstrap/05 Modals.md

```diff
@@ -0,0 +1,68 @@
+If you place DevExtreme components in the [Bootstrap modal](https://getbootstrap.com/docs/5.3/components/modal/), the following issues may occur:
+
+- Users cannot insert, update, or delete a [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/) row.
+- Users cannot enter values in the [FilterBuilder](/Documentation/Guide/UI_Components/FilterBuilder/Overview/).
+- Editors in the [DropDownBox](/Documentation/Guide/UI_Components/DropDownBox/Getting_Started_with_DropDownBox/) lose focus.
+- [TextArea](/Documentation/Guide/UI_Components/TextArea/Getting_Started_with_TextArea/) loses focus.
+- DataGrid [ColumnChooser](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columnChooser/) loses focus.
+
+On the page load, Bootstrap places all items inside the modal into a collection and allows to focus only on the items from the collection. DevExtreme component elements can be generated on the fly, so these elements do not get attached to the focusable collection.
```

Comment:
"the page load" - article use is a bit harder to explain here. But it's not a pre-determined page load event, it's not "the page load". It may be "a page load"...  but in essence we talk about something generic, so I would use zero article. 

Once again - we shouldn't say "the modal", because I assume I can have a few of those on my page. 

And then I'm breaking longer sentences into shorter sentences and avoiding passive voice. 

```suggestion
On page load, Bootstrap locates all modals and places their children into special collections. A modal only allows items from its children collection to receive focus. At the same time, DevExtreme components can generate elements on the fly. Such dynamically-generated elements do not get attached to the focusable collection.
```